src/module/skb/event.rs

```diff
@@ -33,7 +33,156 @@ pub(crate) struct SkbEvent {
+            let protocol = match ip.protocol {
+                1 => " ICMP",
+                2 => " IGMP",
+                6 => " TCP",
+                17 => " UDP",
+                41 => " IPv6",
+                47 => " GRE",
+                50 => " ESP",
+                58 => " ICMPv6",
+                94 => " IPIP",
+                _ => "",
+            };
+
+            write!(f, " len {} proto{} ({})", ip.len, protocol, ip.protocol)?;
+        }
+
+        if let Some(tcp) = &self.tcp {
+            let mut flags = Vec::new();
+            if tcp.flags & 1 << 0 != 0 {
+                flags.push('F');
+            }
+            if tcp.flags & 1 << 1 != 0 {
+                flags.push('S');
+            }
+            if tcp.flags & 1 << 2 != 0 {
+                flags.push('R');
+            }
+            if tcp.flags & 1 << 3 != 0 {
+                flags.push('P');
+            }
+            if tcp.flags & 1 << 4 != 0 {
+                flags.push('.');
+            }
+            if tcp.flags & 1 << 5 != 0 {
+                flags.push('U');
+            }
+            write!(f, " flags [{}]", flags.into_iter().collect::<String>())?;
+
+            let len = len - tcp.doff as u16;
+            if len > 0 {
+                write!(f, " seq {}:{}", tcp.seq, tcp.seq + len as u32)?;
+            } else {
+                write!(f, " seq {}", tcp.seq)?;
+            }
+
+            if tcp.flags & 1 << 4 != 0 {
+                write!(f, " ack {}", tcp.ack_seq)?;
+            }
+
+            write!(f, " win {}", tcp.window)?;
+        }
+
+        if let Some(udp) = &self.udp {
+            write!(f, " len {}", udp.len)?;
+        }
+
+        if let Some(icmp) = &self.icmp {
+            // TODO: text version
+            write!(f, " type {} code {}", icmp.r#type, icmp.code)?;
+        }
+
+        if self.meta.is_some() || self.data_ref.is_some() {
+            write!(f, " skb [")?;
+
+            if let Some(meta) = &self.meta {
+                if meta.csum != 0 {
+                    write!(f, "csum {:#x} ", meta.csum)?;
+                }
+                if meta.hash != 0 {
+                    write!(f, "hash {:#x} ", meta.hash)?;
+                }
+                write!(f, "len {} ", meta.len,)?;
+                if meta.data_len != 0 {
+                    write!(f, "data_len {} ", meta.data_len)?;
+                }
+                write!(f, "priority {} ", meta.priority)?;
+            }
+
+            if let Some(dataref) = &self.data_ref {
+                if dataref.nohdr {
+                    write!(f, "nohdr ")?;
+                }
+                if dataref.cloned {
+                    write!(f, "cloned ")?;
+                }
+                if dataref.fclone > 0 {
+                    write!(f, "fclone {} ", dataref.fclone)?;
+                }
+                write!(f, "users {} dataref {}", dataref.users, dataref.dataref)?;
+            }
+
+            write!(f, "]")?;
+        }
+
+        Ok(())
```

Comment:
Essentially this is formatting 100% of the event info. As soon as we add some other event sections this will not fit in a single terminal line, in which case the output starts to get messy.
Do you think it makes sense to make an opinionated split and only print a subset of the fields by default allowing for "-v", "-vv", "-vvv" to increase the information shown?